mbf_abstract_nav/include/mbf_abstract_nav/abstract_controller_execution.h

```diff
@@ -364,6 +376,9 @@ namespace mbf_abstract_nav
 
     //! replaces parameter angle_tolerance_ for the action
     double action_angle_tolerance_;
+
+    int disabled_tolerance_sec_;
```

Comment:
Use double for times; u don't know in advance the usage people will do
And normally you don't add units to a variable name
```suggestion
    double disabled_tolerance_sec_;
```